ops/scripts/aws_config

```diff
@@ -0,0 +1,2 @@
+[profile BobaGCP]
```

Comment:
A bit of a nit, I don't think we necessarily need to include 'Boba'  here, I'd think simply calling this the "GCP" profile would be good enough.  Presumably if someone is overwriting the aws config already, they would simply overwrite this file

Reply:
Sure. I have renamed it.